Framework/Nexus/src/NexusFileIO.cpp

```diff
@@ -1277,8 +1269,7 @@ int getNexusEntryTypes(const std::string &fileName,
   // for each entry found, look for "analysis" or "definition" text data fields
   // and return value plus entry name
   for (auto &entry : entryList) {
-    //
-    stat = NXopengroup(fileH, entry.c_str(), "NXentry");
+    NXopengroup(fileH, entry.c_str(), "NXentry");
```

Comment:
why is the `stat = ` not necessary here?

Reply:
The value is never used. `stat` gets reassigned immediately in the proceeding `while` loop.

Reply:
oh yeah, I see that now :man_facepalming: